cmd/inferTime.go

```diff
@@ -0,0 +1,163 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"time"
+)
+
+/*
+Copyright © 2020 Jean-Marc Meessen, ON4KJM <on4kjm@gmail.com>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+//InferTimeBlock contains the information describing a time gap
+type InferTimeBlock struct {
+	lastRecordedTime time.Time
+	nextValidTime    time.Time
+
+	//Number of records without actual time
+	noTimeCount int
+
+	//First log entry with missing date
+	logFilePosition int
+
+	//Computed time interval (in seconds)
+	deltatime int
+}
+
+//displayTimeGapInfo will print the details stored in an InferTimeBlock
+func (tb *InferTimeBlock) displayTimeGapInfo() string {
+	timeFormat := "2006-01-02 15:04"
```

Comment:
WDYT of defining a constant with this value ?

Reply:
implemented